docs/source/guide/index.rst

```diff
@@ -541,4 +543,57 @@ See the below CSV file example:
     Import status: Publish targets task id is dd1cbf2320ad4df6bda9ca62cdc0ef82
     Import status: task STARTED
     Import status: task SUCCESS
-    Import status: Finished.
\ No newline at end of file
+    Import status: Finished.
+
+
+Key Management (``key``)
+==========================
+
+.. rstuf-cli-key
+
+It executes commands related to cryptographic key management and may be used
+for managing keys in the Repository Service for TUF.
+
+.. code:: shell
+
+    ❯ rstuf key
+
+    Usage: rstuf key [OPTIONS] COMMAND [ARGS]...
+
+    Cryptographic Key Commands
+
+    ╭─ Options ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │ --help  -h    Show this message and exit.                                                                                        │
+    ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+    ╭─ Commands ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │ generate                        Generate cryptographic keys using the `securesystemslib` library                                 │
+    ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+
+Key Generation (``generate``)
+---------------------------
+
+.. rstuf-cli-key-generate
+
+This command will generate cryptographic keys using the ``securesystemslib`` library.
+The user is requested to provide:
+
+1. the key type, from the supported list of encryption algorithms
+
+2. the key's filename, whose path will be the current working directory
+
+3. a password, to encrypt the private key file
+
+After the above procedure, two files, the private and public key
+(e.g., ``id_ed25519`` and ``id_ed25519.pub``), will be generated in the current
+working directory.
+
+The generated keys may be used in the Repository Service for TUF Ceremony
+process, for the online key or the TUF roles' keys (``root``, ``targets``, etc. keys).
+
+.. code:: shell
```

Comment:
It breaks the build 

```shell
/Users/kdearaujo/devel/RSTUF/repository-service-tuf-cli/docs/source/guide/index.rst:593:Could not lex literal_block as "shell". Highlighting skipped.
```

```suggestion
.. code::
```

Reply:
Indeed, although there are other `.. code:: shell` in the code :thinking:. In any case, I'm changing it for now